pkg/pillar/cmd/zedagent/handlecertconfig.go

```diff
@@ -339,32 +339,21 @@ func sendAttestReqProtobuf(attestReq *attest.ZAttestReq, iteration int) {
 	}
 
 	deferKey := "attest:" + zcdevUUID.String()
-	zedcloud.RemoveDeferred(zedcloudCtx, deferKey)
 
-	buf := bytes.NewBuffer(data)
-	size := int64(proto.Size(attestReq))
 	attestURL := zedcloud.URLPathString(serverNameAndPort, zedcloudCtx.V2API,
 		devUUID, "attest")
-	const bailOnHTTPErr = false
-	_, _, rtf, err := zedcloud.SendOnAllIntf(zedcloudCtx, attestURL,
-		size, buf, iteration, bailOnHTTPErr)
```

Comment:
This means the iteration argument to sendAttestReqProtobuf is no longer needed.
Perhaps that is the case for other functions zedagent as well.
Once the unused ones have been removed, then I suspect the ctx.iteration in zedagent will (only?) be used for the sending of metrics. Or will it be used somewhere else?